metals/src/main/scala/scala/meta/internal/metals/DelegateSetting.scala

```diff
@@ -0,0 +1,28 @@
+package scala.meta.internal.metals
+
+import scala.meta.internal.metals.MetalsEnrichments._
+import scala.meta.io.AbsolutePath
+
+object DelegateSetting {
```

Comment:
Could we keep this in the database?

Reply:
The project ref won't have a db, but we can keep it in db of the main project responsible for that project ref. Drawback is that we connect to db in `MetalsLspService`, this happens before the instance is created, so that would have to be changed somehow. I also thought, that making it very "manual" might be useful, since it can be set by hand this way.

Reply:
yep, that makes sense, thanks!